scripts/core/ui/components/PlainTextEditor/PlainTextEditor.tsx

```diff
@@ -143,25 +157,36 @@ export class PlainTextEditor extends React.Component<IProps, IState> {
     }
 
     render() {
-        const classes = `${this.props.classes} ${
-            this.state.hasFocus ? 'focus' : ''
-        } plain-text-editor`;
+        const classes = classNames(
+            'plain-text-editor',
+            this.props.classes,
+            {focus: this.state.hasFocus},
+            {disabled: this.props.disabled},
+        );
 
         let editorState = this.state.editorState;
 
         if (this.state.hasFocus && this.selection) {
-            editorState = EditorState.forceSelection(editorState, this.selection);
+            editorState = EditorState.forceSelection(
+                editorState,
+                this.selection,
+            );
         }
 
         return (
-            <div className={classes}>
+            <div
+                className={classes}
+                onMouseDown={(ev) => ev.stopPropagation()}
```

Comment:
why are you stopping the propagation?

Reply:
We have this editor inside editor3 (to handle media caption input) so clicking on it would bubble up to the editor3 component and both editors would be focused (and typing would update both the input and editor3). It's the same issue we have with table cells, we just make sure the click event doesn't go to the parent

Reply:
I'd be good to add it as a comment :)